Integrations/MimecastV2/MimecastV2.py

```diff
@@ -443,49 +450,163 @@ def create_policy():
         'toValue': to_value
     }
 
-    policy_list = create_policy_request(policy_obj, option)
+    return policy_obj, option
+
+
+def create_policy():
+    headers = ['Policy ID', 'Description', 'Sender', 'Receiver', 'Bidirectional', 'Start', 'End']
+    context = {}
+    policy_args = demisto.args()
+    policy_obj, option = get_arguments_for_policy_command(policy_args)
+    policy_list = create_or_update_policy_request(policy_obj, option)
     policy = policy_list.get('policy')
     policy_id = policy_list.get('id')
-    title = 'Mimecast Create Policy: \n Policy {} Was Created Successfully!'.format(policy_id)
+    title = 'Mimecast Create Policy: \n Policy Was Created Successfully!'
     sender = policy.get('from')
-    reciever = policy.get('to')
-    contents = {
+    receiver = policy.get('to')
+    description = policy.get('description')
+    content = {
         'Policy ID': policy_id,
+        'Description': description,
         'Sender': {
             'Group': sender.get('groupId'),
             'Email Address': sender.get('emailAddress'),
             'Domain': sender.get('emailDomain'),
             'Type': sender.get('type')
         },
-        'Reciever': {
-            'Group': reciever.get('groupId'),
-            'Email Address': reciever.get('emailAddress'),
-            'Domain': reciever.get('domain'),
-            'Type': reciever.get('type')
+        'Receiver': {
```

Comment:
this is breaking BC need to check that the outputs of the integration does not rely on this path

Reply:
done